template_config.yml

```diff
@@ -22,38 +22,49 @@ check_gettext: true
 check_manifest: true
 check_openapi_schema: true
```

Comment:
you have to disable it,
OpenAPI was fixed at pulpcore 3.10
https://github.com/pulp/pulpcore/pull/1065